CONTRIBUTING.md

```diff
@@ -0,0 +1,88 @@
+# Contributing to dash
+
+## Getting Started
+
+Refer to the [readme](README.md) for installation and development instructions.
+
+## Coding Style
+
+Please lint any additions to react components with `pylint` and `flake8`.
+
+## Pull Request Guidelines
+
+Use the [GitHub flow][] when proposing contributions to this repository (i.e. create a feature branch and submit a PR against the master branch).
+
+## Running the Tests
+
+To run the tests, you can use Python's `unittest` module, or a test runner like `nose2`.
+
+To run all of the tests:
+`python -m unittest tests`
+
+Or:
+`nose2 -v`
+
+## Local configuration
+You can configure the test server with the following variables:
+### DASH_TEST_CHROMEPATH
+If you run a special chrome set the path to your chrome binary with this environment variable.
+
+### DASH_TEST_PROCESSES
+If you encounter errors about Multi-server + Multi-processing when running under Python 3 try running the tests with the number of server processes set to 1.
```

Comment:
The tests also won't run on windows. Maybe we could change the `IntegrationTests` in this repo with the one in dash-core-components, it runs threaded instead when windows is detected.